### ISSUE TITLE
Implement and make Python LSF driver handle crashes

### ISSUE THREAD

**Issue by jonathan-eq** [equinor/ert#6590]
When reimplementing drivers and queue in Python instead of C(++), we need the crashes to be handled the same way.

**Comment by xjules** [comment]
Where is the PR? :)

**Comment by kwinkunks** [comment]
:point_up: @xjules There are a couple #6636 and #6643, see just above your comment.

**Comment by kwinkunks** [comment]
@jonathan-eq Should maybe reword the issue since I think you are also implementing the LSF driver in these PRs?

**Comment by jonathan-eq** [comment]
I am mostly reimplementing the error handling, so I guess it is both?
